src/Console/EventStoreWorker.php

```diff
@@ -112,4 +116,22 @@ protected function makeSerializableEvent(EventRecord $event)
 
         return new JsonEventRecord($data);
     }
+
+    protected function mapToLocalEvent($event)
+    {
+        $eventToClass = config('eventstore.event_to_class');
+        $className = $eventToClass ? $eventToClass($event) : $event->getType();
+        $class = config('eventstore.namespace') . '\\' . $className;
+
+        if (! class_exists($class)) return;
```

Comment:
This should be to implement by user - $eventToClass call is enough (similar to how Laravel is doing it). If we're passing `$event` event_to_classname will be:

```
function($event){
    $class = 'App\\Events\\' . $className;
    if (! class_exists($class)) return;

    $reflection = new ReflectionClass($class);
    if (! $reflection->implementsInterface(CouldBeReceived::class)) return;

    return $reflection->newInstanceArgs($event->getData());
}
```

and then mapToLocalEvent is simply:

```
protected function mapToLocalEvent($event)
{
    $eventToClass = config('eventstore.event_to_class');
    $class = $eventToClass ? $eventToClass($event) : [add some conversion method here based upon . being / or something];
    $class->setEventRecord($event);

    return $class
}